.github/pull_request_template.md

```diff
@@ -6,5 +6,6 @@
 - [ ] Update documents if necessary.
 - [ ] Lint and apply fixes to your code by running `lintrunner -a`
 - [ ] Is this a user-facing change? If yes, give a description of this change to be included in the release notes.
+- [ ] Is this PR including examples changes? If yes, please remember to update [example documentation](https://github.com/microsoft/Olive/blob/main/docs/source/examples.md) in a follow-up PR.
```

Comment:
Can we remove `in a follow-up PR`? Better to update the example page with the current pr?

Reply:
Oh, Ic. The must in follow-up PR, as the linter check will fail.